### ISSUE TITLE
Display video size on disk

### ISSUE THREAD

**Issue by BlitDev** [Chocobozzz/PeerTube#5480]
### Describe the problem to be solved

I think that displaying the size of the video and the entire channel will help to better understand the allocation of quota on the account.

### Describe the solution you would like

- Add video size information (**Videos**, **Playlists** tabs)
- Add information about the entire channel size

**Comment by Chocobozzz** [comment]
I think we should display video size on disk in admin users overview table instead.

**Comment by Kinuseka** [comment]
What they probably mean is that the user should be able to see its own video's storage allocation per video, exclude the allocation for the transcoded resolution. This totally makes sense for storage management for users especially when peertube has a limited storage per instance. 

The user can free up space by deleting very large videos instead of guessing which video is the largest based on the length (length =/= size allocation). The user should be able to sort the biggest video to the smallest.

This is a needed functionality for User side.

**Comment by Chocobozzz** [comment]
Implemented in admin in https://github.com/Chocobozzz/PeerTube/commit/0648d578705c795db22b7e614eed43ccb0e03e69

**Comment by BlitDev** [comment]
> Implemented in admin in [0648d57](https://github.com/Chocobozzz/PeerTube/commit/0648d578705c795db22b7e614eed43ccb0e03e69)

Why can't you display video size not only for admins but also for users?

**Comment by Chocobozzz** [comment]
> Why can't you display video size not only for admins but also for users?

What do you mean? It's just more work I haven't worked on yet

**Comment by BlitDev** [comment]
> I think we should display video size on disk in admin users overview table instead.

Oh, I guess I misunderstood.

**Comment by Chocobozzz** [comment]
> Oh, I guess I misunderstood.

No, it's just arguments after my comment that convinced me to also add this feature to user views as well. It's the reason why this issue is not closed :)

**Comment by candideu** [comment]
+1 for this! As more and more instances move to paid subscriptions, it would be helpful to know which videos take up the most space, and could be replaced with a smaller file.
